h/static/styles/partials-v2/_share-widget.scss

```diff
@@ -0,0 +1,152 @@
+// Share widget
+// ---------------
+// The widget that allows social linking and the ability to
+// copy a url for use elsewhere.
+
+
+// container for widget should not be displayed
+.js-share-widget {
```

Comment:
There is an existing `is-hidden-when-loading` CSS class in `_util.scss` that you can use to hide elements on the page before the JS runs. This class is automatically removed once the element has been upgraded.

To use it, all you need to do is:

```diff
--- a/h/templates/activity/search.html.jinja2
+++ b/h/templates/activity/search.html.jinja2
@@ -415,7 +415,7 @@
       {{ user_sidebar(user) }}
     {% endif %}
 
-    <div class="js-share-widget">
+    <div class="js-share-widget is-hidden-when-loading">
       {{ share_widget() }}
     </div>
```